### PR TITLE
records: cms mc 2011 new dataset

### LINKED ISSUE #2421: CMS: missing MC
From @zenaiev: It would be useful to add 
`dataset=/GluGluToHToGG_M-125_7TeV-powheg-pythia6/Fall11-PU_S6_START44_V9B-v1/AODSIM
`
to the 2011 MC open data to complete the H->gg example

### PR DESCRIPTION
**Pull request by heitorPB**
fix #2421

### CODE REVIEW

**Comment by tiborsimko** [comment]
Note: to be merged after #2479 because of categorisation.

File: cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json [hunk 1]
```diff
@@ -43843,5 +43843,107 @@
...
+    "files": [
+      {
+        "checksum": "adler32:0x00f46e64",
+        "description": "GluGluToHToGG_M-125_7TeV-powheg-pythia6 AOD dataset file index (1 of 1) for access to data via CMS virtual machine",
+        "size": 2264,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/MonteCarlo2011/Fall11/GluGluToHToGG_M-125_7TeV-powheg-pythia6/AODSIM/PU_S6_START44_V9B-v1/file-indexes/CMS_MonteCarlo2011_Fall11_GluGluToHToGG_M-125_7TeV-powheg-pythia6_AODSIM_PU_S6_START44_V9B-v1_0000_file_index.json"
+      },
+      {
+        "checksum": "adler32:0x6f328076",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json]: The leading `0x` should be removed.

File: cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json [hunk 2]
```diff
@@ -43843,5 +43843,107 @@
...
+      ]
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "1681",
+    "run_period": [
+      "Run2011A",
+      "Run2011B"
+    ],
+    "system_details": {
+      "global_tag": "START53_LV6A1::All",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json]: Shall we use `_LV6` here as the rest of 2011 datasets?

Comment (heitorPB) [reply cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json]: The recommended one is `START53_LV6A1::All`. See http://opendata.cern.ch/docs/cms-guide-for-condition-database

Comment (tiborsimko) [reply cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json]: Let us update it elsewhere too then...

**Pull request merged**